web/policies/preview_policy.ex

```diff
@@ -1,5 +1,7 @@
 defmodule CodeCorps.PreviewPolicy do
   alias CodeCorps.User
 
-  def create?(%User{} = _user), do: true
+  alias Ecto.Changeset
+
+  def create?(%User{} = user, %Changeset{changes: %{user: %{data: creator}}}), do: user.id == creator.id
```

Comment:
What are we checking here? That the `creator` has changed in our changeset? Not sure I follow what this method is doing.